www/components/itinerum_tutorial/Tutorial.jsx

```diff
@@ -1,11 +1,40 @@
 /*
  * src/app/help-panel/Tutorial.jsx
  */
-
 class TutorialComponent extends React.Component {
+    propTypes: {
+        isValid: React.PropTypes.bool,
+        result: React.PropTypes.object,
+        schema: React.PropTypes.object
+    }
+    constructor(props) {
+      super(props);
+      this.state = {
+        schema: {
+          fname: this.props.schema.fname,
+          lname: this.props.schema.lname
+        }
+      };
+      this.handleChange = this.handleChange.bind(this);
+    }
+    handleChange(event) {
```

Comment:
Use different handlers!

Reply:
Network call: 

Handled by angular side of the app. Where is this located?? net.py?? Python w/ all network calls somewhere?
How to retrieve a JSON from endpoint (first pull request) -> 
Pull down the JSON and store it within the app database? (All happening outside, local database on the phone itself??) 
Eventually JSON becomes the "schema"-> tells the app how to layout all the data
Return data is just the answers. 
Kyle has react.
Model off of existing code.